doc/rbd/iscsi-target-cli-manual-install.rst

```diff
@@ -64,7 +114,17 @@ on each machine that will be a iSCSI gateway:
        > git clone https://github.com/open-iscsi/tcmu-runner
        > cd tcmu-runner
 
-   Ceph iSCSI requires systemd so the following build command must be used:
+   Probably you have an OS with minimal installation (so without compilers).
```

Comment:
Nit: drop first two sentences.

Reply:
Nit: drop "Probably you have an OS with minimal installation (so without compilers). However developers of 'tcmu-runner' already think about this providing a very usefull script to install all the needed compilers."